scene-format.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
We don't need a JSON representation of the schema. Proto file should be a single source of truth and used as a doc and a code generation source.